pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[tool.poetry]
+name = "foursight"
+version = "0.9.0"
+description = "Serverless Chalice Application for Monitoring"
+authors = ["William Ronchetti <wrr33@cornell.edu>"]
```

Comment:
Small matter, but don't you want your Harvard email here?

Reply:
Correct. Will fix